lib/std/crypto/tls/Stream.zig

```diff
@@ -157,7 +157,8 @@ pub fn writeError(self: *Self, err: Alert.Description) tls.Error {
     self.flush() catch {};
 
     self.close();
-    return err.toError();
+    @panic("ohnooo");
```

Comment:
?

Reply:
Temporary debugging measure to get a stack trace on alerts.